docset/winserver2025-ps/storage/Get-PhysicalDisk.md

```diff
@@ -480,46 +527,58 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
-## INPUTS
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see
+[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+## Inputs
 
 ### Microsoft.Management.Infrastructure.CimInstance#ROOT/Microsoft/Windows/Storage/MSFT_StoragePool
-You can use the pipeline operator to pass an MSFT_StoragePool object to the *StoragePool* parameter.
+
+You can use the pipeline operator to pass an **MSFT_StoragePool** object to the **StoragePool**
+parameter.
 
 ### Microsoft.Management.Infrastructure.CimInstance#ROOT/Microsoft/Windows/Storage/MSFT_StorageSubsystem
-You can use the pipeline operator to pass an MSFT_StorageSubsystem object to the *StorageSubsystem* parameter.
+
+You can use the pipeline operator to pass an **MSFT_StorageSubsystem** object to the
+**StorageSubsystem** parameter.
 
 ### Microsoft.Management.Infrastructure.CimInstance#ROOT/Microsoft/Windows/Storage/MSFT_VirtualDisk
-You can use the pipeline operator to pass an MSFT_VirtualDisk object to the *VirtualDisk* parameter.
 
-## OUTPUTS
+You can use the pipeline operator to pass an **MSFT_VirtualDisk** object to the **VirtualDisk**
+parameter.
+
+## Outputs
 
 ### Microsoft.Management.Infrastructure.CimInstance#ROOT/Microsoft/Windows/Storage/MSFT_PhysicalDisk
-The Get-PhysicalDisk cmdlet returns objects that represent physical disks.
 
-## NOTES
+The `Get-PhysicalDisk` cmdlet returns objects that represent physical disks.
+
+## Notes
 
-* When used in Failover Cluster, cmdlets from the Storage module operate on cluster level (all servers in the cluster).
+* When used in Failover Cluster, cmdlets from the Storage module operate on cluster level (all
+  servers in the cluster).
 
-## RELATED LINKS
+## Related links
 
-[Add-PhysicalDisk](./Add-PhysicalDisk.md)
+- [Add-PhysicalDisk](./Add-PhysicalDisk.md)
 
```

Comment:
Don't change the H2 headers and you can use bullet list for related links. Each link must be on a separate line. Both violate the schema.